apps/src/feedback.js

```diff
@@ -987,6 +989,15 @@ FeedbackUtils.prototype.showSimpleDialog = function (Dialog, options) {
   if (options.bodyText) {
     contentDiv.innerHTML += '<p>' + options.bodyText + '</p>';
   }
+  var textBox;
+  if (options.prompt) {
+    textBox = document.createElement('input');
+    textBox.className = "name-textbox";
```

Comment:
I'd actually just do an inline style in the jsx.

My question was unclear, sorry; I meant why'd you call the class what you called it; it's a bit unclear